### PR TITLE
Update service.md

### PR DESCRIPTION
**Pull request by stacykor**
missing update to Sept 22.

## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ x] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/releasenotes/service.md [hunk 1]
```diff
@@ -9,6 +9,16 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 Service Release Notes for the current year are listed here. If you need to see previous Service Release Notes, check the [Archive](docs/releasenotes/archive).
 
+
+---
+## September 22, 2022 (Security)
+
+Collection
+Update - Good news! We've increased the number of Cloud-to-Cloud Sources that customers with paid Sumo Logic subscriptions can have, from 20 to 50. For users of free accounts, the limit remains 20 Cloud-to-Cloud Sources. For more information, see [Cloud-to-Cloud Integration Framework] (/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/).
```
Comment (kimsauce) [inline docs/releasenotes/service.md:17]: @stacykor  fix broken link

**Review by kimsauce — CHANGES_REQUESTED** [review]
Minor change

**Comment by kimsauce** [comment]
@stacykor , @MariePMcGarry is assigned is migrate over August and Sept: https://app.asana.com/0/1183741177809740/1203018922431321.

**Comment by stacykor** [comment]
> @stacykor , @MariePMcGarry is assigned is migrate over August and Sept: https://app.asana.com/0/1183741177809740/120301892243132

@MariePMcGarry is welcome to make any additional changes. I'm sure we'll release a few more things before 9/30.

File: docs/releasenotes/service.md [hunk 2]
```diff
@@ -9,6 +9,16 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 Service Release Notes for the current year are listed here. If you need to see previous Service Release Notes, check the [Archive](docs/releasenotes/archive).
 
+
+---
+## September 22, 2022 (Security)
+
+Collection
+Update - Good news! We've increased the number of Cloud-to-Cloud Sources that customers with paid Sumo Logic subscriptions can have, from 20 to 50. For users of free accounts, the limit remains 20 Cloud-to-Cloud Sources. For more information, see [Cloud-to-Cloud Integration Framework] (/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework).
```
Comment (kimsauce) [inline docs/releasenotes/service.md:17]: Pls fix broken link. Need to close the gap between text and URL:

```suggestion
Update - Good news! We've increased the number of Cloud-to-Cloud Sources that customers with paid Sumo Logic subscriptions can have, from 20 to 50. For users of free accounts, the limit remains 20 Cloud-to-Cloud Sources. For more information, see [Cloud-to-Cloud Integration Framework](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework).
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
To fix the broken link, need to close the gap between the text and URL

**Review by kimsauce — APPROVED** [review]

File: docs/releasenotes/service.md (hunk 1 above)
Comment (kimsauce) [reply docs/releasenotes/service.md]: Hi @stacykor , to merge this PR, you'll also need to click the "Resolve Conversation" button.

**Pull request merged**